contributing.rst

```diff
@@ -221,7 +221,7 @@ You can install the SDK Platform tools (including :command:`adb`) as a `standalo
 
 .. _docs-workflow-setup:
 
-Getting Ready to Work
+Get Ready to Work
```

Comment:
This is an editorial change, not a typo fix. 
Please revert. 

(We should have a discussion about gerunds in titles separately.)